next/components/ui/FormRJSF/GeneratedFormRJSF.tsx

```diff
@@ -0,0 +1,97 @@
+import LeftIcon from '@assets/images/chevron-left.svg'
+import ArrowRightIcon from '@assets/images/forms/arrow-right.svg'
+import { EFormValue } from '@backend/forms'
+import { SectionContainer } from '@bratislava/ui-bratislava/SectionContainer/SectionContainer'
+import { FormValidation, RJSFSchema } from '@rjsf/utils'
+import { useFormStepper } from '@utils/forms'
+import { useTranslation } from 'next-i18next'
+
+import Button from '../../forms/simple-components/Button'
+import FinalStep from '../../forms/steps/FinalStep'
+import StepperView from '../../forms/steps/StepperView'
+import { ThemedForm } from '../../forms/ThemedForm'
+
+interface FormRJSF {
+  eform: EFormValue
+  escapedSlug: string
+  formSlug: string
+}
+
+const GeneratedFormRJSF = ({ eform, escapedSlug, formSlug }: FormRJSF) => {
+  const [t] = useTranslation('forms')
+  const form = useFormStepper(escapedSlug, eform.schema)
+
+  return (
+    <SectionContainer className="pt-14 md:pt-18">
+      {form.isComplete ? (
```

Comment:
afaik the stepper should be showing on the summary as well - I know designs have iterated here so if you've seen it without previously it was likely an error on our side (we apologise) https://www.figma.com/file/FHBgHIHB62YJjPLZcFMQxZ/ESBS---work-in-progress-(Mestské-konto%2C-Maťa)?node-id=560%3A71674&t=FVbZ8smVd12fMAhx-0

the summary should also be visible in the stepper, and "navigable" through stepper (you should be able to jump to summary and see what you've filled in so far)